docs/DesignLanguage.vue

```diff
@@ -300,13 +300,15 @@
                 <h2 ref="ff-text-input"><pre>ff-text-input</pre></h2>
                 <h3>Properties:</h3>
                 <props-table :rows="cGroups['input'].components[0].props"></props-table>
+                <h3>Methods:</h3>
+                <methods-table :rows="cGroups['input'].components[0].methods" @callMethod="this.$refs['text-input']?.[$event]()"></methods-table>
```

Comment:
Implementing the callback inside the methods-table turned out to be a pain, so I settled on emit+handler.